compiler/rustc_resolve/src/lib.rs

```diff
@@ -1439,9 +1392,14 @@ impl<'a> Resolver<'a> {
         Default::default()
     }
 
-    pub fn into_outputs(self) -> ResolverOutputs {
+    pub fn into_outputs(
+        self,
+    ) -> (Definitions, Box<CrateStoreDyn>, ResolverOutputs, ty::ResolverAstLowering) {
```

Comment:
This tuple got so large that it makes sense to turn it into a structure, named `ResolverOutputs` for example :smile: 
The old `ResolverOutputs` could get some more specific name in that case.